File: 1018.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var n1 = lines.shift();
var n = n1; var valor100 = Math.floor(n / 100); var n = n - valor100*100; var valor50 = Math.floor(n / 50); var n = n - valor50*50; var valor20 = Math.floor(n / 20); var n = n - valor20*20; var valor10 = Math.floor(n / 10); var n = n - valor10*10; var valor5 = Math.floor(n / 5); var n = n - valor5*5; var valor2 = Math.floor(n / 2); var n = n - valor2*2; var valor1 = Math.floor(n / 1);
console.log(`${n1}
${valor100} nota(s) de R$ 100,00
${valor50} nota(s) de R$ 50,00
${valor20} nota(s) de R$ 20,00
${valor10} nota(s) de R$ 10,00
${valor5} nota(s) de R$ 5,00
${valor2} nota(s) de R$ 2,00
${valor1} nota(s) de R$ 1,00`);

File: 1018.py
n1 = int(input())
n = n1
valor100 = n // 100
n = n - valor100*100
valor50 = n // 50
n = n - valor50*50
valor20 = n // 20
n = n - valor20*20
valor10 = n // 10
n = n - valor10*10
valor5 = n // 5
n = n - valor5*5
valor2 = n // 2
n = n - valor2*2
valor1 = n // 1
print(f'''{n1}
{valor100} nota(s) de R$ 100,00
{valor50} nota(s) de R$ 50,00
{valor20} nota(s) de R$ 20,00
{valor10} nota(s) de R$ 10,00
{valor5} nota(s) de R$ 5,00
{valor2} nota(s) de R$ 2,00
{valor1} nota(s) de R$ 1,00''')
print(end='')
